.github/workflows/docker.yml

```diff
@@ -63,16 +65,16 @@ jobs:
     strategy:
       matrix:
         include:
-          - sytest_image_tag: bullseye
+          - sytest_image_tag: bookworm
             dockerfile: synapse
-            tags: "matrixdotorg/sytest-synapse:bullseye"
-            build_args: "SYTEST_IMAGE_TAG=bullseye"
+            tags: "matrixdotorg/sytest-synapse:bookworm"
+            build_args: "SYTEST_IMAGE_TAG=bookworm"
```

Comment:
```suggestion
            build_args: |
              SYTEST_IMAGE_TAG=bookworm
              # This is the default from the image
              # PYTHON_VERSION=3.9
```

(more obvious)

Reply:
If showing the default `PYTHON_VERSION` here is more obvious, would it be better to not give it a default at all, and rely on it being set in every CI matrix value?  (This applies to the `POSTGRES_VERSION` build arg too)

Reply:
16383797